eip-5187.md

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
This file should be: `EIPS/eip-5187.md`.

Reply:
Does EIPW not catch this @SamWilsn?  Would be nice if it did.

Reply:
This never made it to `eipw` because the GitHub Action filters out changes to files not under `EIPS/`.

Reply:
> The GitHub Action filters out changes to files not under `EIPS/`.

That seems like a design flaw...

Reply:
I manually fixed the file name.

Reply:
we've made some changes recently, could you help to review? thanks. @MicahZoltu @Pandapip1 @lightclient